.env.backup.20250919_000000

```
@@ -0,0 +1,89 @@
+# Esta API key permite acceso directo a todos los endpoints sin OAuth2
+# ÚSALO SOLO PARA DESARROLLO LOCAL O SCRIPTS AUTOMATIZADOS
+BACKENDBOT_MASTER_API_KEY=bb_master_2024_secure_key_local_only_48732
+
+# 🔐 SEGURIDAD - JWT SECRET KEY
+# Clave secreta para tokens JWT (generada automáticamente)
+BACKENDBOT_SECRET_KEY=bb_jwt_secret_2024_auto_generated_secure_key_32_chars_min
+
+# 🔐 AUTENTICACIÓN - CREDENCIALES POR DEFECTO
+# ⚠️ CAMBIA ESTAS CREDENCIALES EN PRODUCCIÓN
+BACKENDBOT_DEFAULT_USERNAME=admin
+BACKENDBOT_DEFAULT_PASSWORD=backendbot_secure_2024
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The BACKENDBOT_DEFAULT_PASSWORD key should go before the BACKENDBOT_DEFAULT_USERNAME key

(UnorderedKey)